chrome/content/zotero/itemTreeColumns.jsx

```diff
@@ -23,34 +23,37 @@
 	***** END LICENSE BLOCK *****
 */
 
-(function() {
 const React = require('react');
 const Icons = require('components/icons');
 
 /**
- * @type Column {
- * 	dataKey: string,				// Required, see use in ItemTree#_getRowData()
- *
- * 	defaultIn: Set<string>,			// Types of trees the column is default in. Can be [default, feed];
- * 	disabledIn: Set<string>,		// Types of trees where the column is not available
- * 	defaultSort: number				// Default: 1. -1 for descending sort
- *
- * 	flex: number,					// Default: 1. When the column is added to the tree how much space it should occupy as a flex ratio
- * 	width: string,					// A column width instead of flex ratio. See above.
- * 	fixedWidth: boolean				// Default: false. Set to true to disable column resizing
- * 	staticWidth: boolean			// Default: false. Set to true to prevent columns from changing width when
- * 									// the width of the tree increases or decreases
- * 	minWidth: number,				// Override the default [20px] column min-width for resizing
- *
- * 	label: string,					// The column label. Either a string or the id to an i18n string.
- * 	iconLabel: React.Component,		// Set an Icon label instead of a text-based one
- *
- * 	ignoreInColumnPicker: boolean	// Default: false. Set to true to not display in column picker.
- * 	submenu: boolean,				// Default: false. Set to true to display the column in "More Columns" submenu of column picker.
- *
- * 	primary: boolean,				// Should only be one column at the time. Title is the primary column
- * 	zoteroPersist: Set<string>,		// Which column properties should be persisted between zotero close
- * 	}
+ * @typedef ItemTreeColumnOption
```

Comment:
I just noticed it now, but this should be called "ItemTreeColumnOptions".

Reply:
> Ah, but how to name the array of the `ItemTreeColumnOptions`? This used to be `ItemTreeColumnOption` and we could call that `options`. It would be difficult to distinguish

An array of `ItemTreeColumnOptions` would be called `itemTreeColumnOptions` or `itemTreeColumnOptionsArray` in an exceptional case where additional clarity is needed. I understand why you went with `Option`, but there's no way to singularize singulars further, so when `Option` contains multiple options, there's no way to clarify that, whereas when `Options` is actually a collection of options, you can clarify it by calling it `optionsCollection` or similar.